src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import task.TaskList;
+
+/**
+ * Main Duke class
+ */
+public class Duke {
+    private final DukeListener listener;
+
+    /**
+     * Constructor
+     * Instantiates a (saved) task.Task List and a duke.DukeListener for duke.Duke
```

Comment:
Could use `{@link <ClassName>}` to allow JavaDoc to link the class in the generated docs.

Reply:
Wow, I didn't know this! Thanks for the suggestion! 😜